modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -938,14 +938,15 @@ export class Wallet implements IWallet {
         );
       }
 
-      const value = this.spendableBalanceString();
-      if (_.isUndefined(value) || value === '0') {
+      const value = await this.maximumSpendable();
+      const maximumSpendable = new BigNumber(value.maximumSpendable).toString();
+      if (_.isUndefined(value) || maximumSpendable === '0') {
         throw new Error('no funds to sweep');
       }
       (params as any).recipients = [
         {
           address: params.address,
-          amount: value,
+          amount: maximumSpendable,
```

Comment:
you don't need `lodash` to do a simple `!== undefined` check. Also why convert `maximumSpendable` to a string to perform a numerical check? Why not https://mikemcl.github.io/bignumber.js/#isPos ?

Reply:
updated 👍